src/SFA.DAS.FAT.Web/Views/Courses/CourseDetails.cshtml

```diff
@@ -215,11 +248,39 @@
                         </ul>
                     </dd>
                 </div>
+                
+                <div asp-show="@Model.IsFoundationApprenticeship" class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Course apprentices can progress to
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        <details class="govuk-details">
+                            <summary class="govuk-details__summary">
+                                <span class="govuk-details__summary-text">
+                                    Course apprentices can progress to
+                                </span>
+                            </summary>
+                            <div class="govuk-details__text">
+                                
+                                    Foundation apprenticeships cover some knowledge and skills from these courses. If the apprentice progresses to one of these, what theyv'e learnt will be taken off the assessment. They could finish the course in a shorter time.      
+                                    <br/><br/>
+                                    
+                                    <ul class="govuk-list govuk-list--bullet">
+                                        @foreach (var occupation in Model.RelatedOccupations)
+                                        {
+                                            <li>@occupation.Title (Level @occupation.Level)</li>
```

Comment:
This could be done in ViewModel so we can have coverage

Reply:
ok